COPYING.md

```diff
@@ -74,35 +81,36 @@ public/sounds/lisp | [EdinburghCollective](http://lichess.org/@/EdinburghCollect
 <!-- prettier-ignore -->
 Files | Author(s) | License
 --- | --- | ---
-public/logo, public/favicon.ico, public/apple-touch-icon.png | sadsnake1 | Only use to refer to lichess.org
-public/piece/alpha | Eric Bentzen | "free for personal non commercial use" (see [zip](http://www.enpassant.dk/chess/downl/alpha.zip))
-public/piece/chess7 | [Style-7](http://www.styleseven.com/) | "freeware"
-public/piece/companion | David L. Brown | ["freeware"](http://www.enpassant.dk/chess/fonteng.htm#GC)
-public/piece/fantasy | [Maurizio Monge](http://poisson.phc.unipi.it/~monge/chess_art.php) | "give credit in a suitable way"
-public/piece/leipzig | Armando Hernandez Marroquin | ["freeware"](http://www.enpassant.dk/chess/fonteng.htm#LEIPZIG)
-public/piece/spatial | [Maurizio Monge](http://poisson.phc.unipi.it/~monge/chess_art.php) | "give credit in a suitable way"
-public/piece/reillycraig | [Reilly Craig](https://instagram.com/fader_) |
-public/piece/riohacha | |
+public/logo, public/favicon.ico | internal PlayStrategy team | Only use to refer to playstrategy.org
+public/apple-touch-icon.png | sadsnake1 | Only use to refer to lichess.org
+public/piece/chess/alpha | Eric Bentzen | "free for personal non commercial use" (see [zip](http://www.enpassant.dk/chess/downl/alpha.zip))
+public/piece/chess/chess7 | [Style-7](http://www.styleseven.com/) | "freeware"
+public/piece/chess/companion | David L. Brown | ["freeware"](http://www.enpassant.dk/chess/fonteng.htm#GC)
+public/piece/chess/fantasy | [Maurizio Monge](http://poisson.phc.unipi.it/~monge/chess_art.php) | "give credit in a suitable way"
+public/piece/chess/leipzig | Armando Hernandez Marroquin | ["freeware"](http://www.enpassant.dk/chess/fonteng.htm#LEIPZIG)
+public/piece/chess/spatial | [Maurizio Monge](http://poisson.phc.unipi.it/~monge/chess_art.php) | "give credit in a suitable way"
+public/piece/chess/reillycraig | [Reilly Craig](https://instagram.com/fader_) |
+public/piece/chess/riohacha | |
 public/images/staunton/piece/Staunton | [Eden Murs](https://userstyles.org/styles/134558/lichess-pieces-3d-staunton) (?) |
 
 - The other sounds in public/sound.
 - The other artwork in public/images.
 
 ## External services
 
-Lichess as deployed on https://lichess.org/ also uses these external services:
+Playstrategy as deployed on https://playstrategy.org/ also uses these external services:
 
 - [Cloudflare](https://www.cloudflare.com/) to serve static assets, DNS lookups for email validation
 - [prismic.io](https://prismic.io/) for help/documentation pages and the blog
 - [twitch](https://www.twitch.tv/) for featured livestreams
-- [YouTube](https://www.youtube.com) for featured livestreams and the [Video library](https://lichess.org/video)
-- [stripe](https://stripe.com/) and [PayPal](https://www.paypal.com) for [Patron donations](https://lichess.org/patron)
-- [Spreadshirt](https://shop.spreadshirt.com/lichess-org) for the [Swag store](https://lichess.org/swag)
```

Comment:
haha I missed it then 2 weeks ago
maybe you would like to update the routes files as well ?
https://github.com/Mind-Sports-Games/lila/blob/dev/conf/routes#L794

Reply:
good spot, i'll update that now